app/tests/db_test_utils.py

```diff
@@ -0,0 +1,27 @@
+from app.models import TableMetadata, ColumnMetadata
+
+
+def create_table(name):
+    return TableMetadata(
+        name=name, original_file_name=name + ".csv", description=name + " description"
+    )
+
+
+def create_column(name, table):
+    return ColumnMetadata(
+        table_metadata=table,
+        name=name,
+        title=name + "Title",
+        description=name + " description",
+        data_type_id=1,
```

Comment:
```suggestion
        data_type=1,
```
Is it possible to just set this ForeignKey as an int?

Reply:
Yes, it is.